binders/rabbit-binder/spring-cloud-stream-binder-rabbit/src/test/java/org/springframework/cloud/stream/binder/rabbit/RabbitTestContainer.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2022-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.binder.rabbit;
+
+import java.time.Duration;
+
+import org.testcontainers.containers.RabbitMQContainer;
+
+/**
+ * Provides a static {@link RabbitMQContainer} that can be shared across test classes.
+ *
+ * @author Chris Bono
+ */
+public class RabbitTestContainer {
+
+	private static final RabbitMQContainer RABBITMQ;
+	static {
+		String image = "rabbitmq:management";
```

Comment:
I swtiched off the custom Pivotal image that was being used in [AbstractIntegrationTest](https://github.com/spring-cloud/spring-cloud-stream/pull/2358/files#diff-6631b85f70208404bd3e3682dea09de101fc54eb219264abeec90998645fc118L34). Unless there is a glaring reason to use it I would suggest we move to a standard image configured w/ streams enabled (as below). Also note this allows us to `RabbitMQContainer` for port/url access rather than `GenericContainer`.